src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```

Comment:
better multi-line long attributes, even if there are just 2 of them
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
```